Gemfile

```diff
@@ -57,6 +57,7 @@ group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'web-console'
 
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
   # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
   # gem "rack-mini-profiler"
```

Comment:
- Your tests are currently not running because you have not added the right gems. Please fix this 👍 👍. Please make sure that this part of your gem file looks like this 🙏 
```ruby
group :development, :test do
  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
  gem 'debug', platforms: %i[mri mingw x64_mingw]
   gem 'rails-controller-testing'
   gem 'rspec-rails'
end
```

<img width="704" alt="jryjyj" src="https://github.com/abeltsew/my-blog/assets/73607512/838a88c1-ad63-4966-9455-0db66fd51280">